docs/linked-list/Practice-Problems.md

```diff
@@ -0,0 +1,58 @@
+---
+id: Practice-Problems-on Linked-List
+title: Practice Problems
+sidebar_label: Practice Problems
+sidebar_position: 16
+Description: Here are some practice problems for Linked List data structure divided into topic-wise and difficulty wise.
+tags: [DSA, algorithms,linked list, dsa]
+---
+
+### 1. Basic Operations on Linked List
+
+  - [Reverse a Linked List](https://leetcode.com/problems/reverse-linked-list/description/) 
+  - [Middle of the Linked List](https://leetcode.com/problems/middle-of-the-linked-list/description/)   
+  - [Length of Linked List](https://practice.geeksforgeeks.org/problems/find-length-of-loop/1)  
+  - [Delete Node in a Linked List](https://leetcode.com/problems/delete-node-in-a-linked-list/description/)     
+  - [Swap Nodes in Pairs](https://leetcode.com/problems/swap-nodes-in-pairs/description/)  
+  - [Rotate List](https://leetcode.com/problems/rotate-list/description/)  
+
+### 2. Two Pointer Technique
+  
+  - [Linked List Cycle](https://leetcode.com/problems/linked-list-cycle/description/)  
+  - [Intersection of Two Linked Lists](https://leetcode.com/problems/intersection-of-two-linked-lists/description/)  
+  - [Remove N-th Node from End](https://leetcode.com/problems/remove-nth-node-from-end-of-list/description/)  
+  - [Linked List Cycle II](https://leetcode.com/problems/linked-list-cycle-ii/description/)  
+  - [Split Linked List in Parts](https://leetcode.com/problems/split-linked-list-in-parts/description/)  
+
+### 3. Linked List Merging
+
+  - [Merge Two Sorted Lists](https://leetcode.com/problems/merge-two-sorted-lists/description/)    
+  - [Add Two Numbers](https://leetcode.com/problems/add-two-numbers/description/)  
+  - [Merge in Between Linked Lists](https://leetcode.com/problems/merge-in-between-linked-lists/description/)   
+  - [Sort List](https://leetcode.com/problems/sort-list/description/)  
+  - [Merge k Sorted Lists](https://leetcode.com/problems/merge-k-sorted-lists/description/) 
+  
+### 4. Circular Linked List
+  - [Linked List Cycle](https://leetcode.com/problems/linked-list-cycle/description/)
+  - [Split Circular List](https://www.geeksforgeeks.org/problems/split-a-circular-linked-list-into-two-halves/1/)   
+  - [Find the Winner of the Circular Game](https://leetcode.com/problems/find-the-winner-of-the-circular-game/description/)  
+  - [Sorted insert for circular linked list](geeksforgeeks.org/problems/sorted-insert-for-circular-linked-list/1)  
```

Comment:
replace `geeksforgeeks.org/problems/sorted-insert-for-circular-linked-list/1` with `https://geeksforgeeks.org/problems/sorted-insert-for-circular-linked-list/1`